quickstart/faqs.md

```diff
@@ -74,3 +74,44 @@ Mayastor, as any other solution leveraging TCP for network transport, may suffer
 ## Why do Mayastor pods show high levels of CPU utilization when there is little or no I/O being processed?
 
 Mayastor has been designed so as to be able to leverage the peformance capabilities of contemporary high-end solid-state storage devices.  A significant aspect of this is the selection of a polling based I/O service queue, rather than an interrupt driven one.  This minimises the latency introduced into the data path but at the cost of additional CPU utilisation by the "reactor" - the poller operating at the heart of the Mayastor pod.  When Mayastor pods have been deployed to a cluster, it is expected that these daemonset instances will make full utilization of their CPU allocation, even when there is no I/O load on the cluster.  This is simply the poller continuing to operate at full speed, waiting for I/O.  For the same reason, it is recommended that when configuring the CPU resource limits for the Mayastor daemonset, only full, not fractional, CPU limits are set; fractional allocations will also incur additional latency, resulting in a reduction in overall performance potential. The extent to which this performance degradation is noticeable in practice will depend on the performance of the underlying storage in use, as well as whatvever other bottlenecks/constraints may be present in the system as cofigured.
+
+
+## Does the supportability tool expose sensitive data?
+
+The supportability tool generates support bundles which are used for debugging purposes. These bundles are created in response to user invocation of the tool and can be transmitted only by the user.
+Below is the information collected by the supportability tool that might be identified as 'sensitive' based on the organization's data protection/privacy commitments and security policies.
+**Logs:**
+The default installation of Mayastor includes the deployment of a log aggregation subsystem based on Grafana Loki. All the pods deployed in the same namespace as Mayastor and labelled with `openebs.io/logging=true` will have their logs incorporated within this centralized collector. These logs may include the following information:
+- Kubernetes node hostnames
+- IP addresses
+    - container addresses
+- API endpoints
+    - Mayastor
+    - K8s
```

Comment:
Kubernetes

Reply:
Seeing Kubernetes and K8s used interchangeably across multiple places. Do we need to follow some practice?

Reply:
Good point.  My suggestion would be to treat it, i.e. "k8s" as what it is, a form of abbreviation, and follow the grammatical rules for the same.  That is, the first time it appears in a document, it is shown in full and alongside that abbreviation and subsequently only the abbreviation is used.